### PR TITLE
Fixed type hinting

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: src/Cli/Monolog/ConsoleHandler.php [hunk 1]
```diff
@@ -89,7 +85,7 @@ public function close(): void
         parent::close();
     }
 
-    protected function write(LogRecord|array $record): void
+    protected function write(LogRecord $record): void
     {
         $this->output->write((string) $record['formatted']);
```
[the remark this answers is not in the document]
Comment (lyrixx) [reply src/Cli/Monolog/ConsoleHandler.php:90]: indeed 👍🏼 
BTW, I did not "witch to not support the array type" ; You did by removing support for monolog 2.X

[the remark this answers is not in the document]
Comment (SamMousa) [reply src/Cli/Monolog/ConsoleHandler.php]: Oh yeah, could be xD

**Comment by SamMousa** [comment]
Closing and reopening to test if it now builds the phar.

**Comment by SamMousa** [comment]
Hmm, we need to increase our PHPStan level: https://phpstan.org/r/b2865103-7092-4f04-9460-8e3ffbe1eea4

This code is not guaranteed to work since `->formatted`  has type `mixed`, which is not always castable to string. Since this is not a newly introduced bug I'm gonna merge it anyway. Thanks!

**Pull request merged**